src/driver/AmdCompiler.cpp

```diff
@@ -354,7 +374,8 @@ AMDGPUCompiler::AMDGPUCompiler(const std::string& llvmBin_)
     llvmLinkExe(llvmBin + "/llvm-link"),
```

Comment:
Yes we can, llvLinkExe is used for Driver API path only. AMDGPUCompiler class will be refactored after finishing in-process.